regression/jbmc-strings/StringEquals/Test.java

```diff
@@ -15,4 +18,51 @@ else if (i==4)
         else if (i==5)
             assert(s.equals(x));
     }
+
+    public static boolean referenceImplementation(String s, Object o) {
+        if (s == null)
```

Comment:
Note this can't happen since you always precede referenceImplementation with a virtual call against `s`